### ISSUE TITLE
No new official release since 2016

### ISSUE THREAD

**Issue by daisy-insightsoftware** [Tencent/rapidjson#2321]
Snyk can only track official releases. The latest official release is more than 8 years. 
What's the official release process here for this repo? 
Can we request an official release with a specific commit, e.g. fd3dc29a5c2852df569e1ea81dbde2c412ac5051

**Comment by wertzuz** [comment]
Why are there 100 pull-requests and over 600 open issues and so few commits with a company like Tencent behind it.

**Comment by bogaotory** [comment]
`include/rapidjson` on main works fine as library, but yes, a new official release would be appreciated.

**Comment by digiexchris** [comment]
yeah, this needs some attention. there are deprecation issues with c++20.

**Comment by ks1322** [comment]
See https://github.com/Tencent/rapidjson/issues/2284

**Comment by yehudamber** [comment]
Version 1.1.0 doesn't work with GCC 14 or Clang 19. The problem was solved 1 day after the release (3b2441b), but the Debian package still has it. And other packages (like mlpack) depend on it. This is so frustrating.

**Comment by rexkogitans** [comment]
I can just repeat what everyone says in persistent loops. Most Linux distros pick up versions of software packages from GitHub by looking at their release tag. Not setting a new release tag renders any development of RapidJSON useless for most Linux distros.

**Comment by jlskuz** [comment]
Yeah, it would really helpful to get a release. @miloyip I saw you merged some changes recently. Could help with this?

**Comment by xtexx** [comment]
Looking forward to a new release. pls.

**Comment by digiexchris** [comment]
Since we've had no response from them, I've personally switched to https://github.com/nlohmann/json since it has a modern API that doesn't have a problem with C++20/24 and has more recent releases (along with other nice features)

**Comment by flubshi** [comment]
Maybe we should think about making an official fork - sth like RapidJSON-ng.
Does anyone feel called?

**Comment by xtexx** [comment]
I won't have much time contributing to the fork (if there will be one), but happy to package it for a few distros I am using.

**Comment by SpeedyCraftah** [comment]
> Since we've had no response from them, I've personally switched to https://github.com/nlohmann/json since it has a modern API that doesn't have a problem with C++20/24 and has more recent releases (along with other nice features)

Nlohmann JSON is great, but funnily enough I migrated away from it because it's not designed for great performance, it's acceptable but it does too many allocations under the hood, it's designed to be simple and very object orientated, not fast and efficient. It's too bad simdjson doesn't support writing JSON (I believe they're open to it but lack the maintainers, planning time, etc), otherwise I'd completely move away from rapidjson instead of using it alongside simdjson.

Besides, rapidjson is still *sort of* maintained, I see a few commits here now and then with fixes to bugs, so it's safe to assume the master branch is the "release", I have no idea why they're not packaging these commits into releases anymore though. I think a fork that simply packages commits into releases would work wonders.

**Comment by gvollant** [comment]
see https://github.com/Tencent/rapidjson/issues/2202 https://github.com/Tencent/rapidjson/issues/1919 https://github.com/Tencent/rapidjson/issues/1006 https://github.com/Tencent/rapidjson/issues/1130 https://github.com/Tencent/rapidjson/issues/1341 https://github.com/Tencent/rapidjson/issues/1896 https://github.com/Tencent/rapidjson/issues/1911 https://github.com/Tencent/rapidjson/issues/1881 https://github.com/Tencent/rapidjson/issues/1862 https://github.com/Tencent/rapidjson/issues/1820 https://github.com/Tencent/rapidjson/issues/1130 https://github.com/Tencent/rapidjson/issues/1341 https://github.com/Tencent/rapidjson/issues/1896 https://github.com/Tencent/rapidjson/issues/1911 https://github.com/Tencent/rapidjson/issues/1774 https://github.com/Tencent/rapidjson/issues/1998 https://github.com/Tencent/rapidjson/issues/1976 https://github.com/Tencent/rapidjson/issues/2108 https://github.com/Tencent/rapidjson/issues/2238 https://github.com/Tencent/rapidjson/issues/2266 https://github.com/Tencent/rapidjson/issues/2284 https://github.com/Tencent/rapidjson/issues/2321

**Comment by d3matt** [comment]
Probably not a bad idea to mark some of those as duplicates?

**Comment by wertzuz** [comment]
No one ist moderating the github that's the issue.

**Comment by devkaiwang** [comment]
Let's abandon this lib.
